.travis.yml

```diff
@@ -14,7 +14,8 @@ addons:
   postgresql: "9.6"  # note: travis does not yet support 10.x
 
 install:
-  - pip install -r requirements.txt
+  - pip install pipenv
+  - pipenv install --dev --ignore-pipfile --system --deploy
```

Comment:
I'm not sure `--ignore-pipfile` is really necessary since it will anyhow use the information in the lock file but it also can't hurt I suppose.